MsGraphicsPkg/DisplayEngineDxe/FormDisplay.c

```diff
@@ -636,8 +641,13 @@ UiAddMenuOption (
 
   for (Index = 0; Index < Count; Index++) {
     MenuOption = AllocateZeroPool (sizeof (UI_MENU_OPTION));
-    ASSERT (MenuOption);
+    // MU_CHANGE [BEGIN] - CodeQL change
+    if (MenuOption == NULL) {
+      ASSERT (MenuOption);

```

Comment:
nit: since we are changing it anyway, change to usual style of `MenuOption != NULL`?

Reply:
Updated